packages/account/src/Components/trading-assessment/trading-assessment-dropdown.jsx

```diff
@@ -32,26 +33,24 @@ const TradingAssessmentDropdown = ({
             {item_list.map(question => (
                 <Field name={question.form_control} key={question.form_control}>
                     {() => {
+                        const has_input_error = !values[question.form_control];
                         return (
                             <React.Fragment>
                                 <DesktopWrapper>
                                     <Dropdown
-                                        classNameDisplay={classNames(
-                                            'trading-assessment__wrapper__dropdown--mobile--display',
-                                            {
-                                                'trading-frequency--field':
-                                                    question.form_control ===
-                                                        'trading_frequency_financial_instruments' &&
-                                                    ['ID', 'FR'].includes(getLanguage()),
-                                            }
-                                        )}
+                                        classNameDisplay={classNames({
+                                            'trading-frequency--field':
+                                                question.form_control === 'trading_frequency_financial_instruments' &&
+                                                ['ID', 'FR'].includes(getLanguage()),
```

Comment:
Oh my.. why do we need to hard-code these 2 languages here! 
I know it is unrelated to your PR but please fix this 🙏🏼 

cc: @likhith-deriv

Reply:
sure, will refactor it